### ISSUE TITLE
Menu icon in mobile view jumps open->close

### ISSUE THREAD

**Issue by heykapil** [vsoch/tw-jekyll#3]

**Comment by vsoch** [comment]
Yep reproduced! The trick will be comparing with https://fly.io/docs/ and figuring out what I stripped out that removed the functionality. I do remember testing it early on (and I think it worked) so likely I fussed something up.

**Comment by heykapil** [comment]
Our page is missing essential javascript functionality as https://fly.io/docs/ has. This minified javascript file has our required menu toggle, I'm talking about https://fly.io/public/javascripts/docs.js , but it includes several other things like js for algolia search and all.

**Comment by vsoch** [comment]
Thanks that's helpful! I must have removed it and not checked all functionality. I'm at work but will put in a PR later this evening to fix it up.

**Comment by vsoch** [comment]
okay @heykapil please test! https://github.com/vsoch/tw-jekyll/pull/4

**Comment by heykapil** [comment]
Not working. Thanks for trying out ! I have removed extra code from https://fly.io/public/javascripts/docs.js to this https://imkapil.netlify.app/assets/js/menu.js 
This worked for me. Please see it in action from any mobile device. https://imkapil.netlify.app

**Comment by vsoch** [comment]
Thats's weird, because it worked locally for me. Could you please open a PR with your fix for me to test?

**Comment by vsoch** [comment]
Also it works fine in my preview.... https://9-447414597-gh.circle-artifacts.com/0/tw-jekyll/docs/index.html

screen shot after I turned into mobile and clicked the menu. Are you sure you tested the right thing?

![image](https://user-images.githubusercontent.com/814322/158736519-ef52cceb-fc9d-433d-adfd-398606f3a508.png)

**Comment by heykapil** [comment]
Yup, menu is working in this https://9-447414597-gh.circle-artifacts.com/0/tw-jekyll/docs/index.html preview. Also, it is including algolia search engine for fly.io, look how search looks in desktop mode.

![Screenshot_2022-03-17-10-04-10-307_com android chrome](https://user-images.githubusercontent.com/101615305/158738089-14b54064-ee19-44ba-b386-fca185cf3b8e.jpg)

**Comment by vsoch** [comment]
ah that's the reason it's not working! Could you please provide that as clear feedback next time, because I was only looking at the explicit opening /closing of the menu and did not look at the search.

I've updated the PR and here is the new preview: https://10-447414597-gh.circle-artifacts.com/0/tw-jekyll/index.html

**Comment by heykapil** [comment]
Thanks @vsoch , Menu is working. Noticed in preview that menu button when clicked to expand items moves a bit left and when clicked to close menu items, it moves bit right  Similar thing here https://imkapil.netlify.app/
Just this small tweak and this issue can be closed.

**Comment by vsoch** [comment]
I'm going to merge the PR since the issue is already present and the PR is an improvement, and if you have a suggested fix I'd be happy to test it out in another one. Thanks!

**Comment by vsoch** [comment]
It doesn't bother me that much, so I don't see actively working on it for a fix, but I will leave the issue open if others have suggestions. Thanks again!
